fastseq_cli/transformers_generate.py

```diff
@@ -573,6 +581,10 @@ def run_generate():
                         help="The number of independently computed returned sequences for each element in the batch.")
     parser.add_argument("--seed", type=int, default=None, required=False,
                         help="Specify a random seed for initialization")
+    parser.add_argument("--padding_side", type=str, default=None, required=False,
+                        help="Specify which side the tokenizer should pad")
```

Comment:
nit: could you provide the possible values in the help string (e.g., 'left', 'right`)?